gitlab/v4/objects/service_accounts.py

```diff
@@ -0,0 +1,24 @@
+from gitlab.base import RESTManager, RESTObject
+from gitlab.mixins import CreateMixin
+from gitlab.types import RequiredOptional
+
+__all__ = [
+    "GroupServiceAccount",
+    "GroupServiceAccountManager",
+]
+
+
+class GroupServiceAccount(RESTObject):
+    _id_attr = None
```

Comment:
Are you sure this should be `None`? The `id` is returned on create and if we end up supporting PAT endpoints for service accounts, the endpoint should support the `id` attribute I believe. Does that make sense?

https://docs.gitlab.com/ee/api/groups.html#rotate-a-personal-access-token-for-service-account-user

Reply:
will sort that.